queries.sql

```diff
@@ -7,4 +7,37 @@ SELECT date_of_birth from animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+BEGIN TRANSACTION;
+UPDATE animals SET species='unspecified';
+SELECT *FROM animals;
+ROLLBACK;
+SELECT *FROM animals;
+
+BEGIN TRANSACTION;
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+SELECT *FROM animals;
+UPDATE animals SET species='pokemon' where species IS NULL;
+SELECT *FROM animals;
```

Comment:
- [ ] Please make sure to add a `commit` after making the updates to the animals and also make sure to verify the changes before and after adding the commit.